android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -734,6 +755,55 @@ private void finishPlayerInitialization() {
         applyModifiers();
     }
 
+    private void setupPlaybackService() {
+        if (showNotificationControls & player != null) {
+            playbackServiceConnection = new ServiceConnection() {
+                @Override
+                public void onServiceConnected(ComponentName name, IBinder service) {
+                    playbackServiceBinder = (PlaybackServiceBinder) service;
+
+                    try {
+                        playbackServiceBinder.getService().registerPlayer(player);
+                    } catch (Exception e) {
+                        DebugLog.e("ExoPlayer Playback", "Cloud not register ExoPlayer");
+                    }
+                }
+
+                @Override
+                public void onServiceDisconnected(ComponentName name) {
+                    playbackServiceBinder = null;
```

Comment:
can you playbackServiceBinder.getService().unregisterPlayer(player); here ?
I am afraid the service keep holding a player Reference...
Not sure my comment make sens ...